redhat/frr.spec.in

```diff
@@ -774,7 +774,91 @@ sed -i 's/ -M rpki//' %{_sysconfdir}/frr/daemons
 
 
 %changelog
-* Tue Nov  4 2021 Martin Winter <mwinter@opensourcerouting.org> - %{version}
+* Tue Mar  1 2022 Martin Winter <mwinter@opensourcerouting.org> - %{version}
+
+* Tue Mar  1 2022 Jafar Al-Gharaibeh <jafar@atcorp.com> - 8.2
```

Comment:
Seems like we have already 8.2 here `Tue Feb  1 2022 Donatas Abraitis <donatas.abraitis@gmail.com> - 8.2`?

Reply:
Thanks @ton31337, dropped the extra line.